modules/apps/27-interchain-accounts/keeper/handshake.go

```diff
@@ -64,7 +64,7 @@ func (k Keeper) OnChanOpenInit(
 
 	// Claim channel capability passed back by IBC module
 	if err := k.ClaimCapability(ctx, chanCap, host.ChannelCapabilityPath(portID, channelID)); err != nil {
-		return sdkerrors.Wrap(channeltypes.ErrChannelCapabilityNotFound, err.Error())
+		return sdkerrors.Wrap(err, "failed to claim capability")
```

Comment:
We should probably indicate the port and channel id?

Reply:
Done!